src/App.js

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable linebreak-style */
+// last one
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
+import 'mdb-react-ui-kit/dist/css/mdb.min.css';
+import '@fortawesome/fontawesome-free/css/all.min.css';
+import './components/app.css';
+import Navigation from './components/Navbar/Navbar';
+import Hotels from './components/Hotels/Hotels';
+import AddHotelForm from './pages/AddHotel/AddHotelForm';
+import HotelDetails from './components/HotelDetails/HotelDetails';
+import SignInPage from './pages/Signin/SignInPage';
+import SignUpPage from './pages/SignUp/SignUpPage';
+import About from './pages/About/About';
+import Booking from './pages/Booking/Booking';
+import MyBookings from './pages/MyBookings/MyBookings';
+
+function App() {
```

Comment:
- [ ]  Please, kindly refactor this and all other functions in your project to ES6 arrow functions, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered non-best practice.

You can do a search in Vscode for the "function" word, and correct as needed.  Shouldn't take you more than 5-10 min so don't worry:

![image](https://user-images.githubusercontent.com/57421823/207976656-24e1b377-7f51-4020-9ff7-c17d2f173d3c.png)